<launch>

  <node pkg="laser-odom_calibration" name="calib" type="io_test" output="screen">
    <param name="file_path" type="string" value="/home/txcom-ubuntu64/" />
    <param name="bagname" type="string" value="2021-05-17-15-43-37" />
    <param name="laser_topic" type="string" value="scan" />
    <param name="odom_topic" type="string" value="odom" />
  </node>

</launch>
